kore/app/repl/Main.hs

```diff
@@ -192,6 +205,7 @@ mainWithOptions
         , replScript
         , replMode
         , outputFile
+        , koreLogOptions
         }
   = do
     mLogger <- newMVar emptyLogger
```

Comment:
We need the `LogAction` created from `koreLogOptions` to be put in the `MVar` here, instead of `emptyLogger`. Then we should probably rename `emptySwappableLogger` to `initialLogger`.